packages/website/ts/components/portal_menu.tsx

```diff
@@ -50,6 +50,14 @@ export class PortalMenu extends React.Component<PortalMenuProps, PortalMenuState
                 >
                     {this.renderMenuItemWithIcon('Trade history', 'zmdi-format-list-bulleted')}
                 </MenuItem>
+                <MenuItem
+                    style={this.props.menuItemStyle}
+                    className="py2"
+                    to={`${WebsitePaths.Portal}/weth`}
+                    onClick={this.props.onClick.bind(this)}
+                >
+                    {this.renderMenuItemWithIcon('ETH wrapper', 'zmdi-circle-o')}
```

Comment:
Prefer `Wrapped ETH` for consistency.

Reply:
Hm... This is the menu item name. It should be a call to action similar to the others: `Generate order`, `Fill order`...